apps/condo/domains/ticket/contexts/ActiveCallContext.tsx

```diff
@@ -0,0 +1,70 @@
+import { notification } from 'antd'
+import isUndefined from 'lodash/isUndefined'
+import { createContext, useContext, useEffect, useState } from 'react'
+
+import { useIntl } from '@open-condo/next/intl'
+
+import { usePostMessageContext } from '@condo/domains/common/components/PostMessageProvider'
+
+interface IActiveCallContext {
+    isCallActive: boolean
+    connectedTickets: Array<string>
+}
+
+const ActiveCallContext = createContext<IActiveCallContext>({
+    isCallActive: false,
+    connectedTickets: [],
+})
+
+const useActiveCall = (): IActiveCallContext => useContext(ActiveCallContext)
+
+const ActiveCallContextProvider = ({ children = {} }) => {
+    const inlt = useIntl()
+    const SavedNotificationMessage = inlt.formatMessage({ id: 'callRecord.savedNotification.message' })
+    const SavedNotificationDescription = inlt.formatMessage({ id: 'callRecord.savedNotification.description' })
+    const SaveErrorNotificationMessage = inlt.formatMessage({ id: 'callRecord.saveErrorNotification.message' })
+    const SaveErrorNotificationDescription = inlt.formatMessage({ id: 'callRecord.saveErrorNotification.description' })
+
+    const { addEventHandler } = usePostMessageContext()
+
+    const [isCallActive, setIsCallActive] = useState(false)
+    const [connectedTickets, setConnectedTickets] = useState([])
+
+    useEffect(() => {
+        if (!isUndefined(window)) {
+            addEventHandler('CondoWebSetActiveCall', '*', ({
```

Comment:
`'*'` means every frame will be able to send a message. Is it ok?

Reply:
i think it's ok